lib/api_browser/app/views/types/home/array.html

```diff
@@ -0,0 +1,25 @@
+<div ng-if="error" class="alert alert-danger">
+  <p>The requested type could not be found.</p>
+</div>
```

Comment:
Don't think we need this error handler here, as it already present in the parent template.